history-server.md

```diff
@@ -0,0 +1,76 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
```

Comment:
we should recommend using KUDO `0.7.5` or higher because it contains several important fixes.

Reply:
Updated the version.